app/src/main/java/me/ccrama/redditslide/Activities/MainActivity.java

```diff
@@ -1529,6 +1523,24 @@ public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which)
                     return true;
                 }
             });
+            LinearLayout multireddits_area = header.findViewById(R.id.multireddits_area);
+
+            multireddits_area.setVisibility(View.GONE);
```

Comment:
You can set the view to be initially gone in the XML, slightly preferable to doing it in code. `android:visibility="gone"`